conformance/pom.xml

```diff
@@ -131,30 +131,21 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <version>${maven.enforcer.version}</version>
+                <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
Let's do this in a separate pull request. This is not part of the current task.

Reply:
Cool, done in PR #258